source/elements/oneDPL/source/parallel_api/buffer_wrappers.rst

```diff
@@ -18,7 +18,7 @@ used as iterators in other contexts, including dereference, as these types do no
 requirements for an iterator.
 
 Buffer Position Objects
------------------------
++++++++++++++++++++++++
```

Comment:
This change demotes this into a subsection, to prevent a second link to the "Buffer wrappers" page in the on-page tables of content. I'm piggy-backing on the unrelated PR, but this is an editorial change, so hopefully nobody objects.